format/testfiles/test_ref_heads.rego.formatted

```diff
@@ -17,3 +17,26 @@ r[x] if x := 10
 p.q.r[x] if x := 10
 
 p.q.r[2] = true
+
+g[h].i[j].k = true
```

Comment:
This looks so wrong but `h` and `j` could be rules defined somewhere in the package, right? 😅 (Not that it matters for formatting)

Reply:
True, this isn't internally consistent; but it's valid Rego, and for this rule we're more interested to see what the formatter does with the redundant `{ true }` body.